### ISSUE TITLE
Zobrazení úryvku při vyhledávání

### ISSUE THREAD

**Issue by zabak** [honza-rychtar/kramerius-web-client#310]
Pokud je výsledkem vyhledávání periodikum, tak byť má jen jeden hit, zobrazí se mi po rozkliknutí nejprve úryvek. U knih se dostanu rovnou do textu. U chráněných nevím co jsem nalezl a jestli je to relevantní.

**Comment by LenkaKravecova** [comment]
Pořád to platí? Buď ten problém nevidím nebo jsem tě nepochopila.

**Comment by zabak** [comment]
Ano. V DK když zadám tento dotaz: [http://www.digitalniknihovna.cz/mzk/search/?q=nutno%20uhradili%20vodem%20u%C5%BEitkov%C3%BDm]
tak se mi zobrazí přehled výsledků. Pokud jsou v seznamu výsledků periodika, tak se nejprve zobrazí přehled výsledků v rámci periodika jako karty s krátkou citací. Pokud je ale výsledkem číslo časopisu nebo monografie, tak se jde rovnou na první výskyt hledaného termínu (zobrazí se stránka). V případě chráněného dokumentu si tak nemůžu udělat představu, jestli je daný výsledek hledání relevantní, jestli jsou v něm například obsažena všechna slova apod. Stejný problém má nový klient, tam je potřeba toto také zadat.

**Comment by LenkaKravecova** [comment]
A máš nějaký návrh, jak by to mohlo vypadat? U neveřejných monografií by byly úryvky a u veřejných ne? To je v podstatě přidání meziúrovně k monografiím. No, nevím.

**Comment by zabak** [comment]
Otázka je, jestli by přehled výsledků s úryvkem neměl být zobrazen vždy. Když je v daném dokumentu jen jeden výsledek, tak může být úryvek hned v přehledu výsledků hledání, nebo se to nebude komplikovat a ten mezikrok se přidá vždy. Jde o to, že když něco hledám, tak je pracnější proklikat x stránek, každou zoomovat apod. Než si zobrazit přehled a mít větší představu o tom co najdu.

Petr
